packages/explorer/src/app/(explorer)/[chainName]/worlds/[worldAddress]/explore/SQLEditor.tsx

```diff
@@ -99,6 +105,18 @@ export function SQLEditor({ table }: Props) {
         </div>
 
         <div className="flex justify-end">
+          {tableData && (
+            <span className="mr-4 flex items-center gap-1.5 text-xs text-white/60">
+              <Tooltip text="Execution time for the SQL query">
+                <span className="flex items-center gap-1.5">
+                  <span className="inline-block h-[6px] w-[6px] animate-pulse rounded-full bg-success" />
+                  <span>{Math.round(tableData.queryDuration)}ms</span>
+                </span>
+              </Tooltip>
+              ·<span> {tableData.rows.length} rows</span>
+            </span>
```

Comment:
instead of a bunch of UI indicators, tooltips, etc. what if this just reads
> 200 rows in 200ms

Reply:
I'm leaning towards leaving it with UI indicators of which there is only one primarily i.e. the blinking green dot to show that the query is running. The goal is to indicate clearly that the results are "live". When merging with `pause/resume` live query PR, I'll also change color to indicate if it's paused as well. I'd love to animate it a little better still but the current solution is probably the best bang for the buck, and will leave more fancy animation as a pleasant side quest.

Reply:
They can work independently but yeah, do need to fix merge conflicts